### DISCUSSION TITLE
How to theme librewolf like firefox?

### DISCUSSION THREAD

**Discussion by sanjyay** [prasanthrangan/hyprdots#1314] [Q&A] [answered]
How to make librewolf have the transparency effect that firefox has?

**Comment by kRHYME7** [comment]
https://wiki.hyprland.org/Configuring/Window-Rules/

Should be helpful in the future. 

Add the new configuration at ``` ~/.config/hypr/userprefs.conf ```

**Comment by ghost** [comment, marked as answer]
This is the window rule we use for Firefox: `windowrulev2 = opacity 0.90 0.90,class:^(firefox)$`

Change firefox to whatever name librewolf uses, you can check with `hyprctl clients`.

Add that to `~/.config/hypr/userprefs.conf` like @kRHYME7 said.

**Reply by sanjyay** [reply]
by using hyprctl clients i got the correct name for LibreWolf. Thank you it worked
